slack-lint/src/test/java/slack/lint/resources/MissingResourceImportAliasDetectorTest.kt

```diff
@@ -100,7 +100,7 @@ class MissingResourceImportAliasDetectorTest : BaseSlackLintTest() {
       .files(
         kotlin(
           """
-          package slack.pkg.subpackage
+          package lint.test.pkg
```

Comment:
Note for reviewers: This is the value returned by `context.project.package` in tests.